doc/Makefile

```diff
@@ -45,13 +48,18 @@ clean:
 	rm -rf generated/*
 	rm -rf modules/*
 
+sync_git:
+	git clone git@github.com:benchopt/benchmark_logreg_l2.git $(BENCH_LOGREG_PATH) || (cd $(BENCH_LOGREG_PATH) ; git pull && cd ../../doc)
```

Comment:
the path is here is relative. should this be avoided ? I always run  `make html` from the doc folder.

I also assumed people use SSH not HTTPS

Reply:
The makefile is in the `doc` folder so I think this is ok.

For the clone, I would use `HTTPS` as this is a public repo, this will make sure this command always work.

Reply:
of course, you're right !